src/firefly/js/ui/CatalogSearchMethodType.jsx

```diff
@@ -410,12 +401,10 @@ function renderTargetPanel(groupKey, searchType) {
                      searchType === SpatialMethod.Box.value ||
                      searchType === SpatialMethod.Elliptical.value);
     return (
-        <div className='intarget'>
+        <Box sx={{height: 80}}>
```

Comment:
FYI, height is a property of Box.  You can use it as
```js
<Box height={80}>
```
https://mui.com/system/properties/

Reply:
Oh ok - thanks for letting me know